wrappers/javascript/react-native/package.json

```diff
@@ -18,13 +18,11 @@
   "files": [
```

Comment:
We should add the `native` directory here? Or is that part of another pr? As you alraedy updated the paths in the cmake / build.gradle files

Reply:
Did we not want to fetch them post install?

Reply:
You're totally right 🤦